recipes/cryptopp/all/conanfile.py

```diff
@@ -36,28 +35,18 @@ class CryptoPPConan(ConanFile):
         "use_openmp": False,
     }
 
+    implements = ["auto_shared_fpic"]
+
     def export_sources(self):
         export_conandata_patches(self)
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
-
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def validate_build(self):
         if is_apple_os(self) and cross_building(self) and Version(self.version) <= "8.6.0":
             # See https://github.com/abdes/cryptopp-cmake/pull/38
             raise ConanInvalidConfiguration("cryptopp 8.6.0 and lower do not support cross-building on Apple platforms")
-    
-    def validate(self):
```

Comment:
This was added in https://github.com/conan-io/conan-center-index/pull/12479 but all my local testing is able to successfully compile. Let's see what the CI has to say about it

Reply:
This seems to actually be creating only static libs regardless! Interesting, will revert and work on this on a follow-up PR